manifests/cli.pp

```diff
@@ -22,9 +22,10 @@
         -> Anchor['jenkins::end']
   }
 
-  $jar = "${jenkins::libdir}/jenkins-cli.jar"
-  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/jenkins-cli.jar"
-  $move_jar = "mv WEB-INF/jenkins-cli.jar ${jar}"
+
+  $jar = "${jenkins::libdir}/cli.jar"
+  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/lib/"
+  $move_jar = "mv WEB-INF/lib/cli-*.jar ${jar}"
```

Comment:
Thinking out loud: would this work and provide more compatibility or only ask for more issues
```suggestion
  $move_jar = "mv WEB-INF/lib/*cli*.jar ${jar}"
```

Reply:
if they decide to add some xyz-cli-xyz.jar we might have problems

Reply:
That's exactly what I meant with `ask for more issues`. I'm fine with the current version, but it deserves a note in README.md

Reply:
i dont think there is a minimum version number which should be mentioned. In old jenkins packages both jenkins-cli and cli-version exist

Reply:
the oldest version i checked was jenkins-2.99-1.1

Reply:
Thanks for checking.